src/emqttd_acl_internal.erl

```diff
@@ -92,10 +77,7 @@ filter(subscribe, {_AllowDeny, _Who, subscribe, _Topics}) ->
 filter(_PubSub, {_AllowDeny, _Who, _, _Topics}) ->
     false.
 
-%%------------------------------------------------------------------------------
 %% @doc Check ACL
-%% @end
-%%------------------------------------------------------------------------------
 -spec check_acl({Client, PubSub, Topic}, State) -> allow | deny | ignore when
```

Comment:
The spec in line 81 uses a record, please define a type for the record and use that instead.